script/generate.rb

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env brew ruby
-directories = ["_data/formula", "api/formula", "formula"]
+require 'cask/cask'
```

Comment:
Maybe move the require directory about the `Cask::Cask.each`?

Reply:
Will do.